ee/tabby-ui/lib/tabby/auth.tsx

```diff
@@ -74,6 +74,13 @@ const saveAuthToken = (authData: AuthData) => {
 }
 const clearAuthToken = () => {
   localStorage.removeItem(AUTH_TOKEN_KEY)
+  window.dispatchEvent(
```

Comment:
do we need to manually trigger this? i thought removeItem will trigger such event automatically?

Reply:
got it - in tabby-ui, which part of code relying on this event? could you point me the link?